test/selenium/configurations.rb

```diff
@@ -127,6 +127,12 @@ def configure_for_chrome_web(args)
   common_configuration_for_web(args)
 end
 
+def configure_for_safari_web(args)
+  common_configuration_for_web(args)
+  # Do not block pop-up windows
+  `defaults write com.apple.Safari com.apple.Safari.ContentPageGroupIdentifier.WebKit2JavaScriptCanOpenWindowsAutomatically -bool true`
```

Comment:
The defaults command in OSX let users access the Mac OS X user defaults system. This can be seen at https://developer.apple.com/library/mac/documentation/Darwin/Reference/ManPages/man1/defaults.1.html
More about the terminal command can be seen at https://macmule.com/2012/07/31/disabling-safari-5-1-xs-6-xs-pop-up-blocker-from-terminal-2/